homeassistant/components/cert_expiry/strings.json

```diff
@@ -20,5 +20,12 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_service%]",
       "import_failed": "Import from config failed"
     }
+  },
+  "entity": {
+    "sensor": {
+      "certificate_expiry_timestamp": {
+        "name": "Certificate expiry timestamp"
```

Comment:
The original name had `Cert` because otherwise the name became too long, especially if device name is prefixed.

Also, I think that we can drop `timestamp`, that's obvious from the value.